helper-cli/src/main/kotlin/HelperMain.kt

```diff
@@ -92,6 +93,7 @@ internal class HelperMain : CliktCommand(name = ORTH_NAME, epilog = "* denotes r
             PackageConfigurationCommand(),
             PackageCurationsCommand(),
             RepositoryConfigurationCommand(),
+            SaveResultWithResolvedScopesCommand(),
```

Comment:
Doesn't the existing `SetDependencyRepresentationCommand` do the same thing?

Reply:
Maybe, at least the [`TargetFormat`](https://github.com/oss-review-toolkit/ort/blob/268cf7f127ccfcabf03b0085abe879042a18a7e1/helper-cli/src/main/kotlin/commands/SetDependencyRepresentationCommand.kt#L62-L73) there only supports `GRAPH`. @oheger-bosch Can you tell us more about this? Was this command supposed to also be able to convert to the tree format, or does it do this already and I miss something?

Reply:
The `SetDependencyRepresentationCommand` was indeed intended to do conversions in both directions - therefore the generic name and the `TargetFormat` enumeration. The conversion to the tree format just has not yet been implemented. I think, I started once with an implementation, but did not finish it; I will see whether I can activate this old branch.

Reply:
See #4949.